fbw-common/src/systems/navdata/shared/types/Airport.ts

```diff
@@ -14,7 +14,7 @@ export interface Airport extends DatabaseItem<SectionCode.Airport> {
   /**
    * Airport reference location, and elevation
    */
-  location: ElevatedCoordinates;
+  location: Coordinates & { alt?: number };
```

Comment:
This now has no unit docs on `alt` (don't know if the old one did either, tbh). But why remove this type?